### PR TITLE
Add patch-package to web dev-deps

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

Needed for #3382 - but just going to use `npx patch-package` on release branch

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/ios/Podfile.lock [hunk 1]
```diff
@@ -1202,4 +1202,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 6927948487d04eb4c86fe5056d0c6dc0bd531bf3
 
-COCOAPODS: 1.12.0
+COCOAPODS: 1.12.1
```
Comment (raymondjacobson) [inline packages/mobile/ios/Podfile.lock:1205]: fancy

**Review by raymondjacobson — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-hotfix-patch-package

**Review by dylanjeffers — CHANGES_REQUESTED** [review]
lots o changes, mind if i try my hand at it?

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-hotfix-patch-package

**Pull request closed without merging**